website/content/docs/api-gateway/consul-api-gateway-install.mdx

```diff
@@ -305,6 +305,48 @@ spec:
 
 </CodeBlockConfig>
 
+If you wish to create a route for a `backendRef` in a different namespace, you will need to additionally
+create a [ReferencePolicy](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.ReferencePolicy).
+
+The following example creates a route named `example-route` in namespace `gateway-namespace` with a `backendRef` in namespace `service-namespace` Traffic is allowed because of the `ReferencePolicy` named `reference-policy` in namespace `service-namespace` that allows traffic from `HTTPRoutes` in `gateway-namespace` to `Services` in `service-namespace`.
```

Comment:
```suggestion
The following example creates a route named `example-route` in namespace `gateway-namespace`. This route has a `backendRef` in namespace `service-namespace`. Traffic is allowed because the `ReferencePolicy`, named `reference-policy` in namespace `service-namespace`, allows traffic from `HTTPRoutes` in `gateway-namespace` to `Services` in `service-namespace`.
```

Breaking this single sentences into multiple sentences to make it easier to understand the relationships betweent he similarly named routes and namespaces.

Reply:
I think the suggestion is failing to apply because it was based on the state of things prior to https://github.com/hashicorp/consul/pull/12861/commits/b22db7a1155025ae3200da94ee2d179ac9ed1974

Reply:
Redid this change in a new commit below.